### PR TITLE
feat(across-relayer): sort by deposit size when disputing

### PR DESCRIPTION
**Pull request by chrismaree**
**Motivation**

The disputer bot should dispute the largest invalid relays first to de-risk the system as much as posible.

**Summary**

Updates the L1 client to sort relays by deposit size, when working with the disputer.

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [ ]  Ran end-to-end test, running the code as in production
- [X]  New unit tests created
- [ ]  Existing tests adequate, no new tests required
- [X]  All existing tests pass
- [ ]  Untested

### CODE REVIEW

File: packages/insured-bridge-relayer/test/Relayer.ts [hunk 1]
```diff
@@ -1476,6 +1476,86 @@ describe("Relayer.ts", function () {
...
       assert.equal(disputeEvents.length, 1);
     });
+    it("Always disputes the largest relay first", async function () {
+      // When there are multiple relays with the same amount, the bot should always dispute the largest relay first.
+      const quoteTime = Number((await web3.eth.getBlock("latest")).timestamp) + 60;
+      // Do 3 relays, the middle relay should be the largest. The disputer should dispute this first. Note that there
+      // is no associated L2 deposit so all three are disputable.
+      await l1Token.methods.mint(l1Owner, toBN(depositAmount).muln(4)).send({ from: l1Owner });
+      await l1Token.methods.approve(bridgePool.options.address, toBN(depositAmount).muln(4)).send({ from: l1Owner });
+      console.log("a");
```
Comment (nicholaspai) [inline packages/insured-bridge-relayer/test/Relayer.ts:1486]: left over  console.logs

File: packages/insured-bridge-relayer/test/Relayer.ts [hunk 2]
```diff
@@ -1476,6 +1476,86 @@ describe("Relayer.ts", function () {
...
+            l1Recipient: l2Depositor,
+            amount: depositAmount,
+            slowRelayFeePct: defaultSlowRelayFeePct,
+            instantRelayFeePct: defaultInstantRelayFeePct,
+            quoteTimestamp: quoteTime,
+          },
+          calculateRealizedLpFeePct(rateModel, toBNWei("0"), toBNWei("0.01"))
+        )
+        .send({ from: l1Owner });
+      console.log("b");
```
Comment (nicholaspai) [inline packages/insured-bridge-relayer/test/Relayer.ts:1502]: same

**Review by nicholaspai — COMMENTED** [review]
Great test case and simple change to the client this is awesome! I'll approve once you remove the leftover console.logs

File: packages/insured-bridge-relayer/test/Relayer.ts (hunk 1 above)
Comment (chrismaree) [reply packages/insured-bridge-relayer/test/Relayer.ts]: removed.

**Review by nicholaspai — APPROVED** [review]
Love this PR as it's small but very impactful. Lgtm

**Pull request merged**